habitat_baselines/rl/ppo/policy.py

```diff
@@ -216,11 +218,14 @@ def __init__(
                 EEStartSensorConfig(),
                 EETargetSensorConfig(),
                 JointSensorConfig(),
+                RobotEESensorConfig(),
+                StepCountSensorConfig(),
             ]
         }
         for k, v in self.ssc_dict.items():
             if k in self.observation_space.spaces:
                 self._n_state += v.shape
+                print(f"adding sensor {k} : {v.shape} ")
```

Comment:
Should user logger

Reply:
I don't think logger is important for this research code.